### PR TITLE
Significant reorg.

### PR DESCRIPTION
**Pull request by aphillips**
- Fix CLDR reference from https -> http
- Moved section 3 (language) after intro (so now it's section 2)
- Moved "canonical unicode locale identifier" after "unicode locale"
- Reformatted and edited section on Unicode locales.

***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/ltli/pull/22.html" title="Last updated on Sep 29, 2020, 3:39 PM UTC (46e4b10)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/ltli/22/f7695e6...aphillips:46e4b10.html" title="Last updated on Sep 29, 2020, 3:39 PM UTC (46e4b10)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -130,14 +244,32 @@ <h3>Locales and Internationalization</h3>
...
+        <p class="definition"><dfn data-lt="unicode locale|unicode locale identifier|unicode locale identifiers|unicode locales">Unicode Locale Identifier</dfn> or <em>Unicode Locale</em>. A <a>language tag</a> that follows the additional processing rules defined by [[CLDR]] in UTR#35 [[LDML]]. A Unicode Locale can include a combination of certain language tag extensions ([[RFC6067]], [[RFC6497]]), although it is not required to do so.</p>
+        
+        <p>A Unicode locale provides the ability to specify in a <a>language tag</a> some <a>international preference</a> variations that go beyond linguistic or regional variation or to select formatting behavior or content when there are multiple options or user preferences within a given locale.  Unicode locale identifiers are well-formed BCP47 language tags. [[CLDR]] also specifies some additional rules about the structure and content of the Unicode Locale's language tag as well as supplying specific interpretation of certain subtags. See <a href="https://www.unicode.org/reports/tr35/tr35.html#Unicode_locale_identifier">Section 3.2</a> of [[LDML]] for details. </p>
```
Comment (himorin) [inline index.html]: BCP47 -> [[BCP47]] (just to add a link)

File: index.html [hunk 2]
```diff
@@ -81,7 +81,121 @@ <h2>Introduction</h2>
...
+        
+        <p class="definition"><dfn data-lt="natural language|language">Natural Language</dfn> (or, in this document, just <em>language</em>). The spoken, written, or signed communications used by human beings.</p>
+        
+        <p>There are many ways that languages might be identified and many reasons that software might need to identify the language of content on the Web. Document formats and protocols on the Web generally use the identifiers used in most other parts of the Internet, consisting of the language tags defined in [[BCP47]].</p>
+        
+        <p>[[BCP47]] is a multipart document consisting, at the time this document was published, of two separate RFCs. The first part, called <em>Tags for Identifying Languages</em> [[RFC5646]], defines the grammar, form, and terminology of language tags. The second part, called <em>Matching of Language Tags</em> [[RFC4647]], describes several schemes for atching, comparing, and selecting content using language tags and includes useful terminology related to comparison of language preferences to tagged content.</p>
```
Comment (himorin) [inline index.html]: describes several schemes for atching -> matching (last word)?

Comment (aphillips) [reply index.html]: Fixed.

File: index.html (hunk 1 above)
Comment (aphillips) [reply index.html]: Fixed.

**Comment by r12a** [comment]
I'm going to merge this, because i want to submit a bunch of review comments via a PR, but will develop conflicts if they aren't based on this.  We can still raise comments against the text later, if there are any.

**Comment by fsasaki** [comment]
+1, thanks @r12a  for the merge - that will be easier to review.

**Pull request merged**